4.0.2/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:wheezy
+
+MAINTAINER Jo Shields <jo.shields@xamarin.com>
+
+#based on dockerfile by Michael Friis <friism@gmail.com>
+
+RUN apt-get update \
+	&& apt-get install -y curl \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN apt-key adv --keyserver pgp.mit.edu --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
```

Comment:
We switched from pgp.mit.edu to the ubuntu keyservers in the [website docs](http://www.mono-project.com/docs/getting-started/install/linux/) since the MIT one was often unreliable, we should make the same switch here (though it doesn't impact the docker image much, since the keys are embedded).